index.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Review code</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <h1 id="title">Hello World ! , We are Software developers</h1>
+    <p id="description">
+      Thank you for taking the time to help us improve the platform
+    </p>
+
+    <form id="survey-form">
+      <div class="form-group">
+        <label id="name-label">Name</label>
+        <input type="text" id="name" placeholder="name here" />
+      </div>
+      <div class="form-group">
+        <label id="email-label">Email</label>
+        <input type="email" id="email" placeholder="email" required />
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">Age</label>
+        <input
+          type="number"
+          id="number"
+          placeholder="age"
+          min="0"
+          max="9"
+          required
+        />
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">
+          Which option best describes your current role?</label
+        >
+        <select name="dropdown" id="dropdown">
+          <option value="man">man</option>
+          <option value="girl">girl</option>
+          <option value="boy">boy</option>
+        </select>
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">
+          Would you recommend freeCodeCamp to a friend?</label
+        >
+        <label>
+          <input name="user-recommend" value="Definitely" type="checkbox" />
+          Definitely
+        </label>
+        <label>
+          <input name="user-recommend" value="Maybe" type="checkbox" />
+          Maybe
+        </label>
+        <label>
+          <input name="user-recommend" value="Not sure" type="checkbox" />
+          Not sure
+        </label>
+      </div>
+
+      <div class="form-group">
+        <label class="number-label">
+          What would you like to see improved? (Check all that apply)</label
+        >
+        <label>
+          <input name="prefer" value="Front-end Projects" type="radio" />
+          Front-end Projects
+        </label>
+        <label>
+          <input name="prefer" value="Back-end Projects" type="radio" />
+          Back-end Projects
+        </label>
```

Comment:
Also, 💡  I think you should make use of the formatting option in your editor. This will help to arrange to format your code in a nicer way that will make it easier for other engineers to read your code.